src/main/java/Steps/MyStepdefs.java

```diff
@@ -92,20 +95,19 @@ public void userUpdatesPhoneWithTheValue(String phone) throws Throwable {
     public void userClicksSave() throws Throwable {
        updateForm.saveButtonClick();
         WebDriverWait wait = new WebDriverWait(driver,10);
-        wait.until(ExpectedConditions.urlToBe("https://seleniumclass.000webhostapp.com/web/index.php?r=users%2Fview&id=23"));
+        wait.until(ExpectedConditions.urlToBe("https://seleniumclass.000webhostapp.com/web/index.php?r=users%2Fview&id=" + myUser.id));
     }
 
     @Then("^Data changed$")
     public void dataChanged() throws Throwable {
     driver.manage().timeouts().implicitlyWait(10,TimeUnit.SECONDS);
         HttpResponse<JsonNode> response
-                = Unirest.get("https://seleniumclass.000webhostapp.com/api/v1/users/23").asJson();
+                = Unirest.get("https://seleniumclass.000webhostapp.com/api/v1/users/" + myUser.id).asJson();
 
-        Gson gson = new Gson();
-        Post myUser1 = gson.fromJson(response.getBody().toString(), Post.class);
+        User myUser1 = gson.fromJson(response.getBody().toString(), User.class);
         String myUserString = myUser.name+myUser.username+myUser.address+myUser.email+myUser.phone;
         String myUser1String = myUser1.name+myUser1.username+myUser1.address+myUser1.email+myUser1.phone;
-        Assert.assertNotEquals(myUserString,myUser1String);
+        Assert.assertNotEquals(myUserString, myUser1String);
```

Comment:
I would recommend to create few assertion here.
Assert.assertNotEquals(username1, username2);
Assert.assertNotEquals(name1, name2);
etc.